src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/OptionsTests.cs

```diff
@@ -337,13 +337,16 @@ private static void GenericObjectOrJsonElementConverterTestHelper<T>(string conv
         {
             var options = new JsonSerializerOptions();
 
+            // Initialize the built-in converters.
+            JsonSerializer.Serialize("", options);
```

Comment:
Can revert this diff after https://github.com/dotnet/runtime/pull/51897 is merged.